challenge/windows/post_install.ps1

```diff
@@ -25,6 +25,4 @@ cl challenge-proxy.c
 
 
 # -- shutdown --
-Set-Service -Name sshd -StartupType Manual
-Set-Service -Name tvnserver -StartupType Manual
```

Comment:
"The fix is to set the service type to Manual which means the service will only start if involved to start."

To make sure I understand these changes, and to document them here, can you please clarify this. It looks like in the past it was already "Manual". Instead you are moving this from `post_install.ps1` to `setup.ps1`. Do we have a good understanding of when `tvnserver` (and `sshd`) need to start in order to guarantee the hacker user doesn't have admin privileges? What was going wrong before, and why does this fix it? Again, I'd like to document this since I have a very weak mental model about what is going on here.

Reply:
My bad, typo here: "The fix is to set the service type to Manual which means the service will only start if ~involved~ invoked to start."

The two `Set-Service` lines in `post_install.ps1` changed the two services' type to Manual but it does not persist after reboot, so `tvnserver` will still start Windows boot since its service type is still Automatic.  I haven't looked into why the changes did not persist in `post_install.ps1` yet, so I just moved them to `setup.ps1` which persists after reboot. Also, the `sshd` service's type by default is already Manual,  so the `Service -Name sshd -StartupType Manual` line is not needed, it's just there to be verbose about it needs to be Manual.

We want the `tvnserver` and `sshd` to start after the logout in `startup.ps1`, it guarantees the hacker user doesn't have admin privileges because if the hacker user is currently logged in and group change is made on the hacker user, a logout is needed to reflect that change.  

https://github.com/pwncollege/dojo/blob/661322679a27b3d43cbf2596395615d6339ed8b6/challenge/windows/startup.ps1#L39-L42

What's wrong before is `Start-Service tvnserver`  is not actually controlling when to start the `tvnserver` service since its type is Automatic and the `tvnserver` service will just start with Windows boot. The fix is to change it to Manual and make sure it persists after reboot, then the `tvnserver` service will start when `Start-Service tvnserver` is called. 

Issue before and order of events:
1. Windows boot up
2. `tvnserver` service start (right after boot up and the user can connect right here before )
3. `startup-script.ps1` runs
4. `sshd` service start (at the end of `startup-script.ps1`)

Current (this PR) and order of events:
1. Windows boot up
2. `startup-script.ps1` runs
3. `tvnserver` service start (at the end of `startup-script.ps1`)
4. `sshd` service start (at the end of `startup-script.ps1`)

Reply:
I should have mentioned this earlier but the changes don't persist because because for some reason the actual service creation happens asynchronously, so where the `Set-Service` used to be it would happen too early and just silently error. You can see the error message if you're watching the logs as it builds